Content.Server/Teleportation/TeleportLocationsSystem.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Content.Server.Chat.Systems;
+using Content.Server.Warps;
+using Content.Shared.Teleportation;
+using Content.Shared.Teleportation.Components;
+using Content.Shared.Teleportation.Systems;
+
+namespace Content.Server.Teleportation;
+
+public sealed partial class TeleportLocationsSystem : SharedTeleportLocationsSystem
```

Comment:
Please document your code.

Reply:
Why does this need to be documented? It's on the component.